sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Processor/PartitionLoadBalancer.cs

```diff
@@ -167,9 +167,10 @@ protected PartitionLoadBalancer()
                 cancellationToken.ThrowIfCancellationRequested<TaskCanceledException>();
 
                 // If ownership list retrieval fails, give up on the current cycle.  There's nothing more we can do
-                // without an updated ownership list.
+                // without an updated ownership list.  Set the EventHubName to null so it doesn't modify the exception
+                // message.
```

Comment:
Not sure that I understand why this is a concern.

Reply:
Any exception encountered in the `PartitionLoadBalancer` class is wrapped in an `EventHubsException` before being thrown, so we pass the operation string as the error message. The processor catches this exception, unwraps it and passes it to the error handler.

In this point of the code we are wrapping the exception. If we set the event hub name, the error message will be: `$"{ Operation } ({ EventHubName })"`, and we won't be able to capture the operation string in the format we want. Other places in the code are simply `${ Operation }`.

Reply:
Fair enough.   Can we just call out that we're expecting the raw operation in the comment?   Otherwise, I'm likely to forget again.

Reply:
Just updated the comments.